iib/workers/tasks/utils.py

```diff
@@ -50,6 +121,136 @@ def get_binary_image_from_config(ocp_version, distribution_scope, binary_image_c
     return binary_image
 
 
+def get_bundle_json(bundles):
+    """
+    Get bundle json from grpcurl response.
+
+    :param list bundles: response from grpcurl call to retrieve list of bundles in
+        an index
+    """
+    return [json.loads(bundle) for bundle in re.split(r'(?<=})\n(?={)', bundles)]
+
+
+def _requires_max_ocp_version(bundle):
```

Comment:
Could you move this in front of the function ` add_max_ocp_version_property`? It should be defined before the function is used.

Reply:
May I ask why? I don't think we need to do that in Python unless I am missing something

Reply:
I would say best practices and my editor complains a lot :smile: